CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ project(lokinet
 if(APPLE)
     # Apple build number: must be incremented to submit a new build for the same lokinet version,
     # should be reset to 0 when the lokinet version increments.
-    set(LOKINET_APPLE_BUILD 0)
+    set(LOKINET_APPLE_BUILD 1)
```

Comment:
probably wise to wrap this in an `if(NOT LOKINET_APPLE_BUILD)` so you can override it when building

Reply:
I thought about that, but the reason I didn't is that it's effectively like a version -- if you need to bump the apple build then that needs to get tracked (and possibly bumped again in the future), and when you bump the main version you want to reset this to 0 -- which is why this is closed to the main Lokinet versions in the first place.  I'm not seeing a case where you would want to bump it locally but *not* commit that into the repo.